scripts/generate_schemas.py

```diff
@@ -277,9 +297,35 @@ def convertAvro(self, avdlFile):
             print("converting", avdlFile)
         if self.verbosity > 1:
             print("running:", " ".join(args))
-        # TODO silence the Java output unless we are in verbose mode by
-        # capturing stdout/stderr here
-        subprocess.check_call(args)
+        if self.verbosity > 1:
+            subprocess.check_call(args)
+        else:
+            devnull = open(os.devnull, 'w')
+            subprocess.check_call(args, stdout=devnull, stderr=devnull)
+            devnull.close()
+
+    def extract_post_methods(self, avdlFile):
```

Comment:
We're standardising on camelCase throughout the codebase. Personally I prefer underscore delimited like this, but since the protocol is defined using camelCase, it seemed the lesser of two evils to at least have a consistent system.